### ISSUE TITLE
[BUG] App logo smaller than other ones

### ISSUE THREAD

**Issue by mgenware** [YehudaKremer/msix#179]
I've already had the `logo_path` point to a 1024x1024 png file.

Most app logos in windows 11 store are like this:
![image](https://user-images.githubusercontent.com/11565096/219608928-d50338a3-e11a-44df-b1e6-a8b9b3b9dcfc.png)

Apps built with this plugin are like:
![image](https://user-images.githubusercontent.com/11565096/219609133-e1824ac3-3671-47f8-8753-0a588e1d8f47.png)

My config:
```yaml
msix_config:
  display_name: -
  publisher_display_name: -
  publisher: -
  identity_name: -
  msix_version: 2.0.0.0
  logo_path: .\icon.png
  languages: en
  assets_directory_path: res
```

**Comment by Tienisto** [comment]
I have noticed this too but it worked in the past.

I think `v3.8.0` uses a new major `image v4` version which causes this issue.

**Comment by YehudaKremer** [comment]
Hello

Thank you for reporting this.

i tried to comparer images generated with version 3.7 and version 3.8, but i cant find any differences:
![image](https://user-images.githubusercontent.com/946652/224502172-1ed80f5c-b576-4d21-a665-529547f48285.png)

After checking the code i decided to remove padding from the icons as a fix for this problem: https://github.com/YehudaKremer/msix/commit/ad6725048a8b208bbae9387b6744ea8e0fa5e8f4

I publish new version (3.9.0) with this fix,
please check this version icons when/if you publish your app to the store.

Thank you again ❤️

**Comment by Tienisto** [comment]
Sadly, it didn't fix the problem.

I just tested this. And indeed, the icon in `v3.7.0` was sharp. In `v3.8.0` and `v3.9.0` the icon seems pixelated.

If you need a reproducable app, then you can compile the msix from https://github.com/localsend/localsend

In the following screenshot, you can see the app icons when you open the start menu. The middle icon is based on `v3.7.0`, while the right icon is based on `v3.9.0`.

![grafik](https://user-images.githubusercontent.com/38380847/224519103-d47c07b0-d524-41ed-b1b1-9f479d488ee1.png)

**Comment by YehudaKremer** [comment]
Hello @Tienisto 

My comparison between 3.7.0 and 3.8.0 was wrong ( I forgot to remove the caret symbol `~3.7.0` 😅) .

So now I can see that with `image` v4.0 our code gives a pixelated icon,
to resolve it I added this line https://github.com/YehudaKremer/msix/commit/2c98c2c209cadef8ab69fc44fdce29c967b8cda5#diff-ff7495e8d38f507e530a89e55ff8f5f01c0beb5f53ddca45b1311325420b892dR128 that I found in `image` repo issues: https://github.com/brendan-duncan/image/issues/452#issuecomment-1376695478

I published a new version (3.9.1) with this fix, please see if it solves the pixelated icon problem.

Thank you for reporting, please keep me updated ❤️

**Comment by Tienisto** [comment]
Thank you! You can close this issue now.
